packages/chains/chains/280.ts

```diff
@@ -47,5 +49,5 @@ export default {
   "slip44": 1,
   "slug": "zksync-era-goerli-testnet-deprecated",
   "status": "deprecated",
-  "testnet": true
+  "testnet": false
```

Comment:
@nachoiacovino can u help me fix this by chance?

Reply:
should be testnet:true as well as deprecated state

Reply:
updated them all on retool